engine/lib/membrane_rtc_engine/endpoint.ex

```diff
@@ -55,12 +55,20 @@ defmodule Membrane.RTC.Engine.Endpoint do
     do:
       update_in(endpoint, [:inbound_tracks, track_id, :metadata], fn _old_metadata -> metadata end)
 
-  @spec get_active_track_metadata(endpoint :: t()) :: %{Track.id() => any}
-  def get_active_track_metadata(%__MODULE__{} = endpoint),
+  @spec get_active_tracks(endpoint :: t()) :: %{Track.id() => Track.t()}
+  def get_active_tracks(%__MODULE__{} = endpoint),
     do:
       endpoint.inbound_tracks
       |> Map.values()
       |> Enum.filter(& &1.active?)
+
+  def get_active_tracks(_endpoint), do: %{}
```

Comment:
I agree that we should always pass `Endpoint` struct, but firstly it tried to make it consistent with the current API where `get_active_track_metadata` has a clause where a passed argument is anything besides `Endpont` struct. But since you mentioned it, I can try to change it.